contracts/modules/licensing/LicensingModule.sol

```diff
@@ -382,6 +382,61 @@ contract LicensingModule is
         }
     }
 
+    /// @notice pre-compute the minting license fee for the given IP and license terms.
+    /// the function can be used to calculate the minting license fee before minting license tokens.
+    /// @param licensorIpId The IP ID of the licensor.
+    /// @param licenseTemplate The address of the license template.
+    /// @param licenseTermsId The ID of the license terms.
+    /// @param amount The amount of license tokens to mint.
+    /// @param receiver The address of the receiver.
+    /// @param royaltyContext The context of the royalty.
```

Comment:
Same here, please add return values in the comments.

Reply:
Added comments.